packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -4406,4 +4433,92 @@ export default class Meeting extends StatelessWebexPlugin {
       }
     );
   }
+
+
+  /**
+   *
+   * Handle logging various things for meetings. Such as audio, video and device details.
+   * @returns {Promise} resolves to an array of DeviceInfo
+   * @memberof Meetings
+   */
+  handleMediaLogging = async () => {
+    LoggerProxy.logger.log('Meeting:index#handleMediaLogging');
+    await this.handleDeviceLogging();
+  }
+
+  /**
+   * Handle audio player logging, dumps details such as autoGainControl, echoCancellation
+   * volume and deviceId
+   * @returns {Promise} resolves empty promise
+   * @memberof Meetings
+   */
+  handleAudioLogging = async () => {
```

Comment:
would move this logic to meeting utils as people can access this function

Reply:
Utility file here https://github.com/webex/webex-js-sdk/blob/master/packages/node_modules/%40webex/plugin-meetings/src/meeting/util.js